views/files/files-grid.hbs

```diff
@@ -24,47 +26,55 @@
 							<div class="card-text">
 							  <small class="file-location">{{#if this.context}}{{$t "files.text.foundIn"}} {{this.context}}{{/if}}</small><br>
 								{{#ifeq this.securityCheck.status "blocked"}}
-									<a href="#"
-									class="btn-file-danger"
-									title="{{$t "files.label.dangerousCode"}}">
+									<button href="#"
+									class="file-btn btn-file-danger"
+									title="{{$t "files.label.dangerousCode"}}"
+									aria-label="{{$t "files.label.dangerousCode"}}">
 										<i class="fa fa-exclamation-triangle"></i>
-									</a>
+									</button>
 								{{else}}
-									<a href="/files/file?download=true&file={{../this._id}}&name={{../this.storageFileName}}"
+									<button href="/files/file?download=true&file={{../this._id}}&name={{../this.storageFileName}}"
 										target="_blank"
+										class="file-btn"
 										data-method="download"
-										title="{{$t "files.label.downloadFile"}}">
+										title="{{$t "files.label.downloadFile"}}"
+										aria-label="{{$t "files.label.downloadFile"}}">
 										<i class="fa fa-cloud-download"></i>
-									</a>
+									</button>
 								{{/ifeq}}
 
-								<i  data-file-id="{{this._id}}"
+								<button  href="#"	
+								    data-file-id="{{this._id}}"
 									data-file-name="{{this.name}}"
 									data-file-path="{{this.path}}"
 									title="{{$t "files.label.renameFile"}}"
-									class="fa fa-edit file-name-edit">
-								</i>
+									aria-label="{{$t "files.label.renameFile"}}"
+									class="file-btn fa fa-edit file-name-edit">
+								</button>	
 
 								{{#ifneq this.context 'geteilte Datei'}}
 								{{#userHasPermission "FILE_DELETE"}}
-								<a href="/files/file/"
+								<button href="/files/file/"
 								   target="_blank"
+								   class="file-btn"
 								   data-method="delete"
 								   title="{{$t "files.label.deleteFile"}}"
+								   aria-label="{{$t "files.label.deleteFile"}}"
 								   data-file-id="{{../this._id}}"
-								   data-file-name="{{../this.name}}"
+								   data-file-name="{{../this.name}}">
 									<i class="fa fa-trash-o"></i>
-								</a>
+								</button>
 								{{/userHasPermission}}
 
-								<a href="#"
-								class="btn-file-share"
+								<button href="#"
```

Comment:
we don't need an href + aria-label because it is the same as the title, you can also hide the icon